requirements/tests.txt

```diff
@@ -8,5 +8,6 @@ prompt-toolkit>=2.0
 pygments>=2.2
 codecov
 coverage
-black
+black==19.10b0 --pre
```

Comment:
Same, can this be loosened?

Reply:
same reason here. If you check the CI, you could see the version change breaks the formatting. This is put to match the CI's version.

Reply:
Ahh yeah, I guess I was hoping for a <=19.10b0

Reply:
nope that would break as well. If anyone is to have a black version of 18. something. 

also the semantic versioning is not followed by black either. so it will be hard to pin relatively.  this is a development dependency, I was having failing CI jobs, because I have accidentally had latest version installed with the requirements file.

Reply:
My concern is that I don't know enough about mypy to update this number, if that makes sense. Maybe there is someone else on @xonsh/xore or you can reassure me about maintaining it?

Reply:
it is already part of CI where it was mentioning the version. Also here we have a conversation on using poetry for this project. 
https://github.com/xonsh/xonsh/issues/3825

then it will be very much easier. Also I assure you that I will help you maintaining the versions.